examples/odroid-c2/files/system/oem/01_layout.yaml

```diff
@@ -0,0 +1,9 @@
+name: "Immutable Layout"
+stages:
+  rootfs:
+    - if: '[ ! -f "/run/cos/recovery_mode" ]'
```

Comment:
Maybe a comment here on `why` we are doing this would be good for future us.

Reply:
Addressed in https://github.com/rancher-sandbox/cOS-toolkit/pull/885/commits/3ba3bf43dcb161304218e48129044cdf21c97997 !